docs/deployment/upgrade.md

```diff
@@ -8,53 +8,60 @@ hide_title: true
 
 After the initial deployment, any future updates to the Cumulus deployment from configuration files, Terraform files (`*.tf`), or modules from a new version of Cumulus can be deployed and will update the appropriate portions of the stack as needed.
 
-## Cumulus versioning
+## Cumulus Versioning
 
-Cumulus uses a global versioning approach, meaning version numbers are consistent across all packages and tasks, and semantic versioning to track major, minor, and patch version (i.e. 1.0.0). We use Lerna to manage versioning.
+Cumulus uses a global versioning approach, meaning version numbers are consistent across all packages and tasks, and semantic versioning to track major, minor, and patch version (e.g., 1.0.0). We use Lerna to manage versioning.
```

Comment:
I realize this isn't a new content update, that said: 


` meaning version numbers are consistent across all packages and tasks,`

I'm a little concerned about this - this is only true for our @cumulus/package npm package.    Users might infer *other* packages (e.g. CMA client libs, cumulus-ecs-task, etc ) are also synced which is most definitely not true.  

I'm wondering if we can be explicit here in what we mean when we say packages and tasks somehow...  I'd suggest a link to the NPM repository, but I'm advocating we take down *all* of our tasks from npm publication and probably a good number of our packages as well. 

More to the point, unless they're writing lambdas users shouldn't ever need to care about this -- they're *only* pulling in the TF module at this point. 


I think the requirements in 1967 (All modules should be on the same version - i.e. data-persistence and cumulus) speak to this a little.     I'm torn though, as we're currently publishing those modules, it's just users shouldn't be using them at all anymore outside of node task/core development, for the most part.

Reply:
I agree that we need to update this to make more sense in the Terraform world. Since we're talking about deployment updates, we probably don't need to mention the packages too much.

Reply:
I also think we can take out the sentence `We use Lerna to manage versioning.`

Reply:
I'll take out the bit about Lerna.

Regarding wording around version number consistency, how about something like this:

> meaning version numbers are consistent across all Terraform modules

Reply:
👍